src/notification.scss

```diff
@@ -29,315 +31,235 @@ $block: #{$fd-namespace}-notification;
   $fd-notification-indicator-border-color-success: var(--sapSuccessBorderColor) !default;
   $fd-notification-indicator-border-color-information: var(--sapInformationBorderColor) !default;
 
-  $fd-notification-title-color: var(--sapGroup_TitleTextColor) !default;
-  $fd-notification-header-border-color: var(--sapPageHeader_BorderColor) !default;
-  $fd-notification-content-text-color: var(--sapContent_LabelColor) !default;
-
   // mixins
+
   @mixin fd-priority-indicator($type) {
-    margin: 1px 0 0 0;
-    padding: 1rem 0.5rem 0 1rem;
+    @include fd-icon("message-#{$type}");
+  }
 
-    @include fd-rtl() {
-      padding: 1rem 1rem 0 0.5rem;
+  @mixin fd-truncate {
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+
+  @mixin fd-hide-on-mobile {
+    @include fd-screen(s) {
+      display: none !important;
     }
 
-    @include fd-icon("message-#{$type}");
+    @include fd-screen(m) {
+      display: inline-block !important;
+    }
   }
 
-  // ------------------------ CONTAINER ------------------------
+
   @include fd-reset();
 
-  max-width: $fd-notification-max-width;
   background-color: $fd-notification-background-color;
   box-shadow: $fd-notification-box-shadow;
   border-radius: $fd-notification-border-radius;
-  border-left: 0.375rem solid;
-  border-left-color: $fd-notification-indicator-border-color-default;
-  margin-bottom: 0.5rem;
+  margin-bottom: 1.25rem;
+  width: 100%;
+  max-width: $fd-notification-max-width-m;
+  box-sizing: border-box;
+  padding: 1rem 0.5rem 1rem 1rem;
 
   @include fd-rtl() {
-    border-left: none;
-    border-right: 0.375rem solid;
-    border-right-color: $fd-notification-indicator-border-color-default;
+    padding-right: 1rem;
+    padding-left: 0.5rem;
   }
 
-  // ------------------------ HEADER ------------------------
-  &__header {
+  &__body {
     @include fd-reset();
 
     display: flex;
-    box-shadow: inset 0 -0.0625rem $fd-notification-header-border-color;
+    justify-content: space-between;
+
+    &--message {
+      padding-top: 1rem;
+    }
   }
 
-  &__indicator {
+  &__group {
     @include fd-reset();
 
-    &--error {
-      @include fd-priority-indicator('error');
+    overflow: hidden;
 
-      color: $fd-notification-indicator-color-error;
-    }
+    .#{$block}__body {
+      padding: 1rem 0.5rem 1rem 1rem;
+      border-bottom: $fd-notification-border;
 
-    &--success {
-      @include fd-priority-indicator('success');
+      &:last-child {
+        border-bottom: none;
+      }
 
-      color: $fd-notification-indicator-color-success;
+      @include fd-rtl() {
+        padding-left: 0.5rem;
+        padding-right: 1rem;
```

Comment:
same for these. All paddings/margins that are used in RTL should be in variables. This way if in future the values change we won't update them in multiple places and less chance for forgetting to update the RTL values